### DISCUSSION TITLE
Embracing CodeBase if committing to Web3?

### DISCUSSION THREAD

**Discussion by koalazub** [pkgxdev/pkgx#289] [Ideas]
Hey! I'm pretty excited about this project and looking at giving it a whirl over the weekend. I also wanted to ask if you've considered embracing Web3 at it's source and whether you've looked into maybe collaborating with [CodeBase](https://codebase.org/) to host/mirror your repo? 
I figured it'd help tremendously with really showing what Web3 is supposed to do

**Comment by mxcl** [comment]
GitHub has powerful network effects, and asking devs to learn new tools is a hindrance to adoption. However longer term we would like our source control to be decentralized, definitely. codebase is new to me. I would enjoy a more developer focused intro if there is one you can provide.
